packages/terra-application-links/src/tabs/ApplicationTabs.jsx

```diff
@@ -80,13 +80,11 @@ class ApplicationTabs extends React.Component {
     this.handleResize(this.contentWidth);
   }
 
-  componentWillReceiveProps(newProps) {
```

Comment:
We'll want to look at either using shouldComponentUpdate or moving this check to the render method.

Reply:
Updated this to use shouldComponentUpdate and added a page to manually trigger links length.
Commit: https://github.com/cerner/terra-framework/pull/365/commits/0f1d2006e8e545605f24a8d0c23adca7cf853f19
Test Page: https://terra-framework-deploye-pr-365.herokuapp.com/#/tests/terra-application-links/application-links/application-tabs-length-change